Docker/Dockerfile

```diff
@@ -1,91 +1,65 @@
-    && cmake --build build --target install \
-    && cd .. && rm -rf llvm
+RUN git clone --depth 1 --single-branch --branch release_40 https://github.com/llvm-mirror/llvm.git && \
+    git clone --depth 1 --single-branch --branch release_40 https://github.com/llvm-mirror/clang.git llvm/tools/clang && \
+    cd llvm && cmake -H. -Bbuild -GNinja -DCMAKE_INSTALL_PREFIX=/opt/wasm -DLLVM_TARGETS_TO_BUILD= -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly -DCMAKE_BUILD_TYPE=Release && \
+    cmake --build build --target install && cd .. && rm -rf llvm
 
-RUN wget https://github.com/WebAssembly/binaryen/archive/1.37.21.tar.gz -O - | tar -xz \
-  && cd binaryen-1.37.21 \
-  && cmake -H. -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release \
-  && cmake --build build --target install \
-  && cd .. && rm -rf binaryen-1.37.21
+RUN wget https://github.com/WebAssembly/binaryen/archive/1.37.21.tar.gz -O - | tar -xz && \
+    cd binaryen-1.37.21 && cmake -H. -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release && \
+    cmake --build build --target install && cd .. && rm -rf binaryen-1.37.21
 
+RUN git clone --depth 1 git://github.com/cryptonomex/secp256k1-zkp && \
+    cd secp256k1-zkp && ./autogen.sh && ./configure --prefix=/usr/local && make install && cd .. && rm -rf secp256k1-zkp
 
-RUN git clone --depth 1 git://github.com/cryptonomex/secp256k1-zkp \
-    && cd secp256k1-zkp \
-    && ./autogen.sh \
-    && ./configure --prefix=/usr/local \
-    && make install \
-    && cd .. && rm -rf secp256k1-zkp
-
-RUN git clone --depth 1 -b releases/stable git://github.com/mongodb/mongo-cxx-driver \
-    && cd mongo-cxx-driver \
-    && cmake -H. -Bbuild -G Ninja -DCMAKE_BUILD_TYPE=Release  -DCMAKE_INSTALL_PREFIX=/usr/local\
-    && cmake --build build --target install
-
-
-### If you don't want to change the depedencies, you can commet out above lines and uncoment the following line to get faster build time.
-# FROM huangminghuang/eos_builder as builder
-
-RUN git clone -b master --depth 1 https://github.com/EOSIO/eos.git --recursive \
-    && cd eos \
-    && cmake -H. -B"/tmp/build" -GNinja -DCMAKE_BUILD_TYPE=Release -DWASM_LLVM_CONFIG=/opt/wasm/bin/llvm-config -DCMAKE_CXX_COMPILER=clang++ \
-       -DCMAKE_C_COMPILER=clang -DCMAKE_INSTALL_PREFIX=/opt/eos  -DSecp256k1_ROOT_DIR=/usr/local \
-    && cmake --build /tmp/build --target install
-
+RUN git clone --depth 1 -b releases/stable git://github.com/mongodb/mongo-cxx-driver && \
+    cd mongo-cxx-driver && cmake -H. -Bbuild -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local && \
+    cmake --build build --target install
 
+RUN git clone -b master --depth 1 https://github.com/EOSIO/eos.git --recursive && \
+    cd eos && cmake -H. -B"/tmp/build" -GNinja -DCMAKE_BUILD_TYPE=Release -DWASM_LLVM_CONFIG=/opt/wasm/bin/llvm-config -DCMAKE_CXX_COMPILER=clang++ \
+       -DCMAKE_C_COMPILER=clang -DCMAKE_INSTALL_PREFIX=/opt/eos  -DSecp256k1_ROOT_DIR=/usr/local && \
+    cmake --build /tmp/build --target install
 
```

Comment:
If there is anything here besides reformatting feel free to submit a new pull request.